src/cmap/auth/mongodb_oidc/callback_workflow.ts

```diff
@@ -55,97 +42,50 @@ export class CallbackWorkflow implements Workflow {
     return { speculativeAuthenticate: document };
   }
 
+  /**
+   * Reauthenticate the callback workflow.
+   * For reauthentication:
+   * - Check if the connection's accessToken is not equal to the token manager's.
```

Comment:
from the spec:

> Additionally, drivers MUST cache the access token used to authenticate a connection on the connection object (henceforth referred to as the Connection Cache).

It's not immediately clear how we're satisfying this requirement.  Do we cache the token on the connection anywhere?

Reply:
As discussed the use of a connection cache seemed redundant as the client cache is the source of truth for all connections. It made more sense here to always use the client cache.